lib/systems/parse.nix

```diff
@@ -464,6 +472,8 @@ rec {
             if lib.versionAtLeast (parsed.cpu.version or "0") "6"
             then abis.gnueabihf
             else abis.gnueabi
+          # Default ppc64 BE to ELFv2
+          else if isPower64 parsed && isBigEndian parsed then abis.gnuabielfv2
```

Comment:
Were these two lines really essential?

They cause a problem: now if we parse `powerpc64-apple-linux-gnu` and then unparse it, we get back `powerpc64-apple-linux-gnuabielfv2`.  That disagrees with `gnu-config`:

```
$ /nix/store/rvxxihfvp9z7h4lw6w60lv7mvsn97dwn-gnu-config-2023-01-21/config.sub powerpc64-apple-linux-gnu
powerpc64-apple-linux-gnu
```

I still agree with nixpkgs PowerPC64-BE using elfv2, I just don't think the *triple parsing* routine is the right place to impose that default...

I've included a commit to revert these two lines as part of #235230; if that's a problem, or if there's a different way we should be handling this, please let me know.

Reply:
428c5456060ebe25ac889d9f666dfd3d114cbff0